src/ui/digital-exchange/Sidebar.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import MarketplaceFilterContainer from 'ui/digital-exchange/MarketplaceFilterContainer';
 import CategoryFilterContainer from 'ui/digital-exchange/CategoryFilterContainer';
+import RatingFilterContainer from './RatingFilterContainer';
```

Comment:
Same here: 
```import RatingFilterContainer from 'ui/digital-exchange/RatingFilterContainer';```

Reply:
fixed in commit 94e21ab0189f339d67a26b342b6df9905ab79739